cg/constants/subject.py

```diff
@@ -1,19 +1,17 @@
-from enum import IntEnum, StrEnum
+from enum import IntEnum, StrEnum, auto
 
 
 class Sex(StrEnum):
```

Comment:
```suggestion
class Sex(Enum):
```

Reply:
I think if there is no need to access `str` functions within the enum then there is no need for the `StrEnum` class. Right, @Vince-janv ?